lib/polymorphic_integer_type.rb

```diff
@@ -3,6 +3,8 @@
 require "polymorphic_integer_type/version"
 require "polymorphic_integer_type/extensions"
 require "polymorphic_integer_type/mapping"
+require "polymorphic_integer_type/module_generator"
+require "polymorphic_integer_type/belongs_to_polymorphic_association_extension"
```

Comment:
this require was the heart of the issue, as it was previously tied to an active record version < 5.2.0, but in reality we want it all the time